kubetest/main.go

```diff
@@ -717,6 +717,9 @@ func prepareGcp(o *options) error {
 		os.Setenv("NODE_OS_DISTRIBUTION", nod)
 	}
 	if o.gcpProject == "" {
+		log.Print(`--gcp-project is missing, trying to fetch a project from boskos.
+				  (for local runs please set --gcp-project to your dev project)`)
```

Comment:
won't this print a newline and bunch of whitespace?

Reply:
hummm gofmt allows that